cluster_traffic_capture/cluster_traffic_capture/gen_haproxy_cfg.py

```diff
@@ -71,7 +71,9 @@ def _get_fe_config_haproxy(haproxy_port: int, mirror: bool) -> str:
     declare capture request len 80000
     declare capture response len 80000
     http-request capture req.body id 0
-    log-format Request-URI:\\ %[capture.req.uri]\\nRequest-Method:\\ %[capture.req.method]\\nRequest-Body:\\ %[capture.req.hdr(0)]\\nResponse-Body:\\ %[capture.res.hdr(0)]
+    http-request capture req.hdrs len 512
```

Comment:
Why 512 characters? Is the body included as part of this 512? if so, it seems very possible that the body will push out all of the actual headers

Reply:
I did have it 4096 at one point but didn't think that was necessary, now I think otherwise as I just ran a test and the body is in fact included in as part of the 512. Any suggested length?

Reply:
I don't think we ever want to truncate here.  So, how about ~1MB (1048576 characters) for all the capture mechanisms.  We'll need to either find a better solution long term or scream this limit loud and clear to users.